frontend/tailwind.config.js

```diff
@@ -1,8 +1,18 @@
 /** @type {import('tailwindcss').Config} */
 export default {
-	content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
-	theme: {
-		extend: {},
-	},
-	plugins: [],
+  content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+  darkMode: 'media',
+  theme: {
+    extend: {
+      colors: {
+        dark: {
+          bg: 'var(--bg-dark)',
+          field: 'var(--bg-dark-field)',
+          card: 'var(--bg-dark-card)',
+          text: 'var(--text-dark)',
+        },
```

Comment:
What is the use of using this ? Please justify, so that we know if it actually required

Reply:
sir this help in to give variable or definition to color

Reply:
@SankalpHaritash21, Please work on the variable names as mentioned above, even with css variable
`card: 'var(--bg-dark-card)'` it is saying color of `bg-dark-card` to `text` and you are using this for text color 
chaning it to something like `dark-card-text-color` would be less confusing and easy to interrupt.